inji-web/src/components/Help/HelpAccordion.tsx

```diff
@@ -1,51 +1,129 @@
+            content:createContent("item10",[t("item10.description1")]),
+        },
+        {
+            title: t("item11.title"),
+            content:createContent("item11",[t("item11.description1"), t("item11.description2"), t("item11.description3"), t("item11.description4"), t("item11.description5")]),
+        },
+        {
+            title: t("item12.title"),
+            content:createContent("item12",[t("item12.description1"), t("item12.description2"), t("item12.description3")]),
+        },
+        {
+            title: t("item13.title"),
+            content:createContent("item13",[t("item13.description1"), t("item13.description2"), t("item13.description3")]),
+        },
+        {
+            title: t("item14.title"),
+            content:createContent("item14",[t("item14.description1"), t("item14.description2"), t("item14.description3")]),
+        },
+        {
+            title: t("item15.title"),
+            content:createContent("item15",[t("item15.description1"), t("item15.description2"), t("item15.description3")]),
+        },
+        {
+            title: t("item16.title"),
+            content:createContent("item16",[t("item16.description1"), t("item16.description2"), t("item16.description3")]),
+        },
+        {
+            title: t("item17.title"),
+            content:createContent("item17",[t("item17.description1"), t("item17.description2"), t("item17.description3")]),
+        },
+        {
+            title: t("item18.title"),
+            content:createContent("item18",[t("item18.description1"), t("item18.description2"), t("item18.description3")]),
+        },
+        {
+            title: t("item19.title"),
+            content:createContent("item19",[t("item19.description1"), t("item19.description2"), t("item19.description3")]),
         },
+        {
+            title: t("item20.title"),
+            content:createContent("item20",[t("item20.description1")]),
+        },
+        {
+            title: t("item21.title"),
+            content:createContent("item21",[t("item21.description1")]),
+        },
+        {
+            title: t("item22.title"),
+            content:createContent("item22",[t("item22.description1")]),
+        },
+        {
+            title: t("item23.title"),
+            content:createContent("item23",[t("item23.description1"),t("item23.description2"),t("item23.description3"),t("item23.description4")]),
+        }
     ];
+
     return (
         <React.Fragment>
             <div data-testid="Help-Accordion-Container">
                 {accordionItems.map((item, index) => (
-                    <HelpAccordionItem id={index}
-                                       key={index}
-                                       title={item.title}
-                                       content={item.content}
-                                       open={open}
-                                       setOpen={setOpen}
+                    <HelpAccordionItem
```

Comment:
Seems intendation is changed, keep the intendation consistant, please set the default intendation as your default in the IDE

Reply:
Still Intendation is not changed